### PR TITLE
add podman minitutorial

### PR DESCRIPTION
**Pull request by baude**
Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: README_PODMAN.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Using the dnsname plugin with Podman
+
+The purpose of the *dnsname* plugin is to allow containers to resolve each other by name.  The plugin adds each
+container's name to an instance of a dnsmasq server.  The plugin is enabled through adding it to a network's
+CNI configuration.  The containers will only be able to resolve each other if they are on the same CNI network.
+
+**Note**: This only works with non-rootless containers.
```
Comment (haircommander) [inline README_PODMAN.md]: The double negative is iffy to me.
I'd suggested `this only works with containers spawned by the root user` or something

Comment (rhatdan) [reply README_PODMAN.md]: This does not work with rootless containers.

File: README.md [hunk 2]
```diff
@@ -7,6 +7,9 @@ the pod and its IP address are added to a network specific hosts file that dnsma
 is removed from the network, it will remove the entry from the hosts file.  Each CNI network will have its own dnsmasq
 instance.
 
+The plugin was specifically designed for the [Podman](https://github.com/containers/libpod) container engine.  To use
+*dnsname* with Podman, follow the [mini-tutorial](README_PODMAN.md)
+
```
Comment (rhatdan) [inline README.md]: How about:
The *dnsname* plugin was specifically designed for the [Podman](https://github.com/containers/libpod) container engine.  Follow the **[mini-tutorial](README_PODMAN.md) to** use it with Podman,

File: README_PODMAN.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Using the dnsname plugin with Podman
+
+The purpose of the *dnsname* plugin is to allow containers to resolve each other by name.  The plugin adds each
```
Comment (rhatdan) [inline README_PODMAN.md]: The *dnsname* plufin allows containers to resolve each other by name. The plugin adds ...

File: README_PODMAN.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+```
+The following example [configuration file](example/cni-podman1.conflist) shows a usable example for Podman.
+
+# Example: container name resolution
+
+1. sudo podman run -dt --name web --network foobar quay.io/libpod/alpine_nginx:latest
+    5139d65d22135e9ecab511559d863754550894a32285befd94dab231017048c2
+
+    Note: we use the --network foobar here. Also, in this test image, the nginx server will respond with
```
Comment (rhatdan) [inline README_PODMAN.md]: How do I get this to happen on the `default` network?  Or how do I make `foobar` my default network?

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**